src/commands/CreateCommand.js

```diff
@@ -0,0 +1,33 @@
+import Command from "../Command";
+import ConfigUtilities from "../ConfigUtilities";
+import FileSystemUtilities from "../FileSystemUtilities";
+import NpmUtilities from "../NpmUtilities";
+import path from "path";
+
+export default class LsCommand extends Command {
+  initialize(callback) {
+    this.pkgName = this.input[0];
+
+    callback(null, true);
+  }
+
+  execute(callback) {
+    const author = NpmUtilities.getWhoIAm();
+    const name = this.pkgName;
+    const pkgRoot = path.join(".", "packages", this.pkgName);
+    const registry = ConfigUtilities.read(this.repository.rootPath).registry;
+    FileSystemUtilities.mkdirp(pkgRoot, () => {
+      const manifest = {
+        name,
+        author,
+        version: "1.0.0"
+      };
+      if (registry) {
+        manifest.publishConfig = { registry };
```

Comment:
I wonder if a nicer way to handle a `registry` in Lerna config would be to specify it on the command line when we use npm commands.  That way a _change_ to the registry in `lerna.json` would have an immediate effect across packages.